Add registries settings (#2303)
saucelabs/sauce-docs: docs/web-apps/automated-testing/cucumberjs-playwright/yaml.md

---

> ### `registry`

Reviewer: What about keeping `registry` and just mentioning it's deprecated and using `registries` instead?

<p><small>| OPTIONAL | STRING |</small></p>

:::note

> This setting is supported up to Playwright 1.35.1. For later version, use `registries`.

Reviewer 2: ```suggestion
This setting is supported up to Playwright 1.35.1. For newer versions, use `registries`.
```

---

### `registries`

<p><small>| OPTIONAL | ARRAY |</small></p>

> Specifies the location of the npm registry, scope and credentials. If the registry is inside a private network, you need to establish a tunnel using [Sauce Connect](/dev/cli/saucectl/#run-tests-on-sauce-labs-with-sauce-connect).

Reviewer 3: ```suggestion
Specifies the location of the npm registry, scope, and credentials. If the registry is inside a private network, you must establish a tunnel using [Sauce Connect](/dev/cli/saucectl/#run-tests-on-sauce-labs-with-sauce-connect).
```

> Specifies the location of the npm registry source, scope and credentials. If the registry source is a private address and you are running tests on Sauce Cloud, you can provide access to the registry source using [Sauce Connect](/dev/cli/saucectl/#run-tests-on-sauce-labs-with-sauce-connect).

Reviewer 2: ```suggestion
Specifies the location of the npm registry, scope and credentials. If the registry is inside a private network, you need to establish a tunnel using [Sauce Connect](/dev/cli/saucectl/#run-tests-on-sauce-labs-with-sauce-connect).
```

```yaml
  registries:
    - url: https://registry.npmjs.org
    - url: https://private.registry.company.org
      scope: "@company"
      authToken: secretToken
```

---

#### `url`

Specifies the URL of the npm registry.

<p><small>| REQUIRED | STRING |</small></p>

```yaml
    - url: https://registry.npmjs.org
```

---

#### `scope`

Specifies which scope is associated with this registry.
See [Associating a scope with a registry](https://docs.npmjs.com/cli/v9/using-npm/scope#associating-a-scope-with-a-registry).

<p><small>| OPTIONAL | STRING |</small></p>

```yaml
    - url: https://registry.npmjs.org
      scope: "@company"
```

---

#### `authToken`

> Specifies the authentitcation token to be used with this registry.

Reviewer 3: ```suggestion
Specifies the authentication token to be used with this registry.
```
